### PR TITLE
`world/feature/**` mappings

### PR DESCRIPTION
**Pull request by NoComment1105**
This PR maps all unmapped things in the `net/minecraft/world/feature/**` package and subpackages. Most of it is method and contructor arguments but there are a few methods and fields that got mapped too.

### CODE REVIEW

File: mappings/net/minecraft/world/gen/feature/ConfiguredFeature.mapping [hunk 1]
```diff
@@ -8,6 +8,8 @@ CLASS net/minecraft/unmapped/C_stwyylfs net/minecraft/world/gen/feature/Configur
 	METHOD <init> (Lnet/minecraft/unmapped/C_lowstvki;Lnet/minecraft/unmapped/C_jpnewgxf;)V
 		ARG 1 feature
 		ARG 2 config
+	METHOD equals (Ljava/lang/Object;)Z
```
Comment (IotaBread) [inline mappings/net/minecraft/world/gen/feature/ConfiguredFeature.mapping:10]: `m_ifdiewua` and `m_wrlrbrkd` should be renamed to `feature` and `config` respectively

File: mappings/net/minecraft/world/gen/feature/ConfiguredStructureFeature.mapping [hunk 2]
```diff
@@ -1,9 +1,22 @@
 CLASS net/minecraft/unmapped/C_licgnyqv net/minecraft/world/gen/feature/ConfiguredStructureFeature
```
Comment (IotaBread) [inline mappings/net/minecraft/world/gen/feature/ConfiguredStructureFeature.mapping]: Also mapped in #68

File: mappings/net/minecraft/world/gen/feature/ConfiguredStructureFeatures.mapping [hunk 3]
```diff
@@ -1,2 +1,36 @@
 CLASS net/minecraft/unmapped/C_jjmsgutv net/minecraft/world/gen/feature/ConfiguredStructureFeatures
+	FIELD f_aaxxefnw MONUMENT Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_anbjnztl JUNGLE_PYRAMID Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_apqafzvk DESERT_PYRAMID Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_bwyhbopf VILLAGE_TAIGA Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_dgbzmuae RUINED_PORTAL_MOUNTAIN Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_dygfetkd RUINED_PORTAL_JUNGLE Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_erwqosvs VILLAGE_SNOWY Lnet/minecraft/unmapped/C_cjzoxshv;
+	FIELD f_exalyvfz BURRIED_TREASURE Lnet/minecraft/unmapped/C_cjzoxshv;
```
Comment (IotaBread) [inline mappings/net/minecraft/world/gen/feature/ConfiguredStructureFeatures.mapping]: Typo of buried

File: mappings/net/minecraft/world/gen/feature/PlacedFeature.mapping [hunk 4]
```diff
@@ -1,18 +1,42 @@
...
+		ARG 2 chunkGenerator
+		ARG 3 random
+		ARG 4 pos
+	METHOD m_xpycbbci placeWithContext (Lnet/minecraft/unmapped/C_hszwdqtd;Ljava/util/Random;Lnet/minecraft/unmapped/C_hynzadkk;)Z
+		ARG 1 decoratorContext
+		ARG 2 random
+		ARG 3 pos
 	METHOD m_xzvkggoc (Lcom/mojang/serialization/codecs/RecordCodecBuilder$Instance;)Lcom/mojang/datafixers/kinds/App;
 		ARG 0 instance
+	CLASS C_nehzjnhe Test
```
Comment (IotaBread) [inline mappings/net/minecraft/world/gen/feature/PlacedFeature.mapping]: Should probably note that this class is literally named `test` in mojmap, and isn't used in production anywhere

File: mappings/net/minecraft/world/gen/feature/ConfiguredFeature.mapping (hunk 1 above)
Comment (NoComment1105) [reply mappings/net/minecraft/world/gen/feature/ConfiguredFeature.mapping:10]: Why? This class isn't a record

Comment (IotaBread) [reply mappings/net/minecraft/world/gen/feature/ConfiguredFeature.mapping:10]: Ah, I thought it was one, my bad

Comment (NoComment1105) [reply mappings/net/minecraft/world/gen/feature/ConfiguredFeature.mapping:10]: Ah oki

**Review by OroArmor — APPROVED** [review]

File: mappings/net/minecraft/world/gen/feature/ConfiguredStructureFeatures.mapping [hunk 5]
```diff
@@ -1,2 +1,36 @@
 CLASS net/minecraft/unmapped/C_jjmsgutv net/minecraft/world/gen/feature/ConfiguredStructureFeatures
```
Comment (triphora) [inline mappings/net/minecraft/world/gen/feature/ConfiguredStructureFeatures.mapping]: Aren't all of these mapped in #90?

File: mappings/net/minecraft/world/gen/feature/StructureFeature.mapping [hunk 6]
```diff
@@ -16,8 +20,16 @@ CLASS net/minecraft/unmapped/C_vvmtobpf net/minecraft/world/gen/feature/Structur
...
+	METHOD m_tiobtmtl configured (Lnet/minecraft/unmapped/C_jpnewgxf;Lnet/minecraft/unmapped/C_ednuhnnn;Z)Lnet/minecraft/unmapped/C_licgnyqv;
+	METHOD m_titzgnss canGenerate (Lnet/minecraft/unmapped/C_wqxmvzdq;Lnet/minecraft/unmapped/C_kgfalbim;Lnet/minecraft/unmapped/C_ajguowya;Lnet/minecraft/unmapped/C_rhdulivw;JLnet/minecraft/unmapped/C_ynrszrtu;Lnet/minecraft/unmapped/C_jpnewgxf;Lnet/minecraft/unmapped/C_qpninoyb;Ljava/util/function/Predicate;)Z
+	METHOD m_utcfbjyw configured (Lnet/minecraft/unmapped/C_jpnewgxf;Lnet/minecraft/unmapped/C_ednuhnnn;Ljava/util/Map;)Lnet/minecraft/unmapped/C_licgnyqv;
+	METHOD m_vfjkdycf getPieceGenerator ()Lnet/minecraft/unmapped/C_ukcqthce;
 	METHOD m_xnhhytsz register (Ljava/lang/String;Lnet/minecraft/unmapped/C_vvmtobpf;Lnet/minecraft/unmapped/C_rdrmebyw$C_eibovkzt;)Lnet/minecraft/unmapped/C_vvmtobpf;
 		ARG 0 name
 		ARG 1 structureFeature
 		ARG 2 step
 	METHOD m_ywspjwtx getCodec ()Lcom/mojang/serialization/Codec;
+	METHOD m_zlyoxgdd confirgured (Lnet/minecraft/unmapped/C_jpnewgxf;Lnet/minecraft/unmapped/C_ednuhnnn;)Lnet/minecraft/unmapped/C_licgnyqv;
```
Comment (triphora) [inline mappings/net/minecraft/world/gen/feature/StructureFeature.mapping]:
```suggestion
	METHOD m_zlyoxgdd configured (Lnet/minecraft/unmapped/C_jpnewgxf;Lnet/minecraft/unmapped/C_ednuhnnn;)Lnet/minecraft/unmapped/C_licgnyqv;
```

File: mappings/net/minecraft/world/gen/feature/DripstoneClusterFeatureConfig.mapping [hunk 7]
```diff
@@ -21,5 +21,7 @@ CLASS net/minecraft/unmapped/C_htmabiqc net/minecraft/world/gen/feature/Dripston
 		ARG 7 density
 		ARG 8 wetness
 		ARG 9 wetnessMean
+		ARG 10 maxDistanceFromCenterAffectingChanceOfDripstoneColumn
+		ARG 11 maxDistanceFromCenterAffectingHeightBias
```
Comment (triphora) [inline mappings/net/minecraft/world/gen/feature/DripstoneClusterFeatureConfig.mapping:25]: ...these are pretty bad

File: mappings/net/minecraft/world/gen/feature/ConfiguredStructureFeatures.mapping (hunk 5 above)
Comment (NoComment1105) [reply mappings/net/minecraft/world/gen/feature/ConfiguredStructureFeatures.mapping]: Quite possibly. Let me check

File: mappings/net/minecraft/world/gen/feature/PlacedFeature.mapping (hunk 4 above)
Comment (triphora) [reply mappings/net/minecraft/world/gen/feature/PlacedFeature.mapping]: I swear this is mapped in another PR, but I can't figure out which

**Review by triphora — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/world/gen/feature/DripstoneClusterFeatureConfig.mapping (hunk 7 above)
Comment (NoComment1105) [reply mappings/net/minecraft/world/gen/feature/DripstoneClusterFeatureConfig.mapping:25]: They were already mapped names, I just added the constructor params

File: mappings/net/minecraft/world/gen/feature/ConfiguredStructureFeatures.mapping (hunk 5 above)
Comment (NoComment1105) [reply mappings/net/minecraft/world/gen/feature/ConfiguredStructureFeatures.mapping]: Yes they are

Comment (NoComment1105) [reply mappings/net/minecraft/world/gen/feature/ConfiguredStructureFeatures.mapping]: I'll fix it tomorrow

**Review by IotaBread — APPROVED** [review]

**Review by triphora — APPROVED** [review]

**Review by LambdAurora — APPROVED** [review]

**Pull request merged**